internal/model/notif_telegram.go

```diff
@@ -5,11 +5,12 @@ const NotifTelegramDefaultTemplateBody = `Docker tag {{ if .Entry.Image.HubLink
 
 // NotifTelegram holds Telegram notification configuration details
 type NotifTelegram struct {
-	Token        string   `yaml:"token,omitempty" json:"token,omitempty" validate:"omitempty"`
-	TokenFile    string   `yaml:"tokenFile,omitempty" json:"tokenFile,omitempty" validate:"omitempty,file"`
-	ChatIDs      []string `yaml:"chatIDs,omitempty" json:"chatIDs,omitempty" validate:"omitempty"`
-	ChatIDsFile  string   `yaml:"chatIDsFile,omitempty" json:"chatIDsFile,omitempty" validate:"omitempty,file"`
-	TemplateBody string   `yaml:"templateBody,omitempty" json:"templateBody,omitempty" validate:"required"`
+	Token               string   `yaml:"token,omitempty" json:"token,omitempty" validate:"omitempty"`
+	TokenFile           string   `yaml:"tokenFile,omitempty" json:"tokenFile,omitempty" validate:"omitempty,file"`
+	ChatIDs             []string `yaml:"chatIDs,omitempty" json:"chatIDs,omitempty" validate:"omitempty"`
+	ChatIDsFile         string   `yaml:"chatIDsFile,omitempty" json:"chatIDsFile,omitempty" validate:"omitempty,file"`
+	TemplateBody        string   `yaml:"templateBody,omitempty" json:"templateBody,omitempty" validate:"required"`
+	DisableNotification *bool    `yaml:"disableNotification,omitempty" json:"disableNotification,omitempty" validate:"omitempty"`
```

Comment:
default should be set in https://github.com/crazy-max/diun/blob/4c5e8ac53a9aefe4119ad568e11469c9ba8d6f76/internal/model/notif_discord.go#L27

```go
s.DisableNotification = utl.NewFalse()
```